server/.gitignore

```diff
@@ -0,0 +1,29 @@
+
+@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
```

Comment:
Just wondering... did we introduce yarn as another package manager? We already have npm for this and unless we absolutely need yarn, let's just stick with npm since they should, in theory, do the same thing. If there's a package available on yarn but not for npm, then we should maybe bring this up during standup and decide if we want to add yarn or look for an alternative to the package.

Resources for why yarn and npm may not work well together
[here](https://stackoverflow.com/questions/49589493/is-there-any-harm-in-using-npm-and-yarn-in-the-same-project) and 
[here](https://www.reddit.com/r/javascript/comments/976dxy/i_have_both_yarn_and_npm_installed_and_theyre/)